app/src/main/java/com/google/maps/android/compose/MapInColumnActivity.kt

```diff
@@ -0,0 +1,224 @@
+                    columnScrollingEnabled = false
+                    Log.d(
+                        TAG,
+                        "User touched map - Disabling column scrolling after user touched this Box..."
+                    )
+                },
+                onMapLoaded = { }
+            )
+        }
+    }
+}
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun MapInColumn(
+    modifier: Modifier = Modifier,
+    cameraPositionState: CameraPositionState,
+    columnScrollingEnabled: Boolean,
+    onMapTouched: () -> Unit,
+    onMapLoaded: () -> Unit,
+) {
+    Surface(
+        modifier = modifier,
+        color = MaterialTheme.colors.background
+    ) {
+        var isMapLoaded by remember { mutableStateOf(false) }
+
+        Column(
+            Modifier
+                .fillMaxSize()
+                .verticalScroll(
+                    rememberScrollState(),
+                    columnScrollingEnabled
+                ),
+            horizontalAlignment = Alignment.Start
+        ) {
+            Spacer(modifier = Modifier.padding(10.dp))
+            for (i in 1..20) {
+                Text(
+                    text = "Item $i",
+                    modifier = Modifier
+                        .padding(start = 10.dp)
+                        .testTag("Item $i")
+                )
+            }
+            Spacer(modifier = Modifier.padding(10.dp))
+
+            Box(
+                Modifier
+                    .fillMaxWidth()
+                    .height(200.dp)
+                    .testTag("Map")
+            ) {
+                GoogleMapViewInColumn(
+                    modifier = Modifier
+                        .fillMaxSize(),
+                    cameraPositionState = cameraPositionState,
+                    onMapLoaded = {
+                        isMapLoaded = true
+                        onMapLoaded()
+                    },
+                )
+                Box(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .pointerInteropFilter(
+                            onTouchEvent = {
+                                when (it.action) {
+                                    MotionEvent.ACTION_DOWN -> {
+                                        onMapTouched()
+                                        false
+                                    }
+                                    else -> {
+                                        Log.d(
+                                            TAG,
+                                            "MotionEvent ${it.action} - this never triggers."
+                                        )
+                                        true
+                                    }
+                                }
+                            }
+                        )
+                )
+                if (!isMapLoaded) {
+                    androidx.compose.animation.AnimatedVisibility(
```

Comment:
Add import statement above so you can use just `AnimatedVisibility` here?

Reply:
When I try this I get a compile error:

```
e: C:\android-projects\android-maps-compose\app\src\main\java\com\google\maps\android\compose\MapInColumnActivity.kt: (159, 21): 'fun ColumnScope.AnimatedVisibility(visible: Boolean, modifier: Modifier = ..., enter: EnterTransition = ..., exit: ExitTransition = ..., label: String = ..., content: AnimatedVisibilityScope.() -> Unit): Unit' can't be called in this context by implicit receiver. Use the explicit one if necessary
```

I ran across this on SO:
https://stackoverflow.com/questions/67975569/why-cant-i-use-animatedvisibility-in-a-boxscope

Apparently it's a bug:
https://youtrack.jetbrains.com/issue/KT-48215